jetstream/cli.py

```diff
@@ -129,6 +171,18 @@ def run(project_id, dataset_id, date, experiment_slug, config_file):
     thrown during some experiment analyses. This ensures that the Airflow task will
     not retry the task and run all of the analyses again.
     """
+    if argo:
+        return submit_workflow(
+            project_id,
+            zone,
+            cluster_id,
+            RUN_WORKFLOW,
+            {"date": date.strftime("%Y-%m-%d")},
+            monitor_status=monitor_status,
+        )
```

Comment:
I added a `--argo` option to some CLI commands. If set, it will use Argo to execute the step. In the case of running Jetstream daily in Airflow this will look like: https://github.com/mozilla/telemetry-airflow/pull/1170

Alternatively, I was also considering having a separate `argo` CLI command group for all the commands only used in the context of Argo:

`jetstream argo run --date=<>`
`jetstream argo rerun --experiment_slug=<>`
`jetstream argo rerun_config_changed`
`jetstream argo analyse_experiment --experiment_config=<>`
`jetstream argo get_active_experiments`